src/ensembl/src/services/analytics-service.ts

```diff
@@ -1,15 +1,18 @@
 import ReactGA from 'react-ga';
-import { AnalyticsType } from 'src/analyticsHelper';
+import { AnalyticsOptions } from 'src/analyticsHelper';
 
 import config from 'config';
 
-const { googleAnalyticsKey = '' } = config;
+const { googleAnalyticsKey = '', isTest } = config;
 
 class AnalyticsTracking {
   private reactGA: typeof ReactGA;
 
   public constructor() {
-    ReactGA.initialize(googleAnalyticsKey);
+    ReactGA.initialize(googleAnalyticsKey, {
```

Comment:
Typescript is upset here:

![image](https://user-images.githubusercontent.com/6834224/61453165-5a12b300-a955-11e9-8b91-a7a185e48a3b.png)

(also, it's generally a good thing to run `npm run check-types` before submitting a PR 😉  )

Reply:
Not sure why it's upset here?
```
export function initialize(trackingCode: string, options?: InitializeOptions): void;
export function initialize(trackers: Tracker[], options?: InitializeOptions): void;
```

Reply:
Interesting. Will `ReactGA.initialize(googleAnalyticsKey as string` help?

Reply:
Tried it. It's still the same.

Reply:
It seems that they overwrite the first export of the `initialize` type with the second export. If I change the order of exports in the def file (which we can't do, because it's third-party), it works.

Only we also need to update our typings/config.d.ts file:

```
declare module 'config' {
  const config: {
    isDevelopment: boolean,
    isProduction: boolean,
    isTest: boolean,

    [key: string]: string
  };
  export = config;
}
```

Reply:
Updated the `typings/config.d.ts` file and this error is now gone 👍